DigitalLearningSolutions.Data/Enums/TrackerEndpointResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DigitalLearningSolutions.Data.Enums
+{
+    using DigitalLearningSolutions.Data.Models.Tracker;
+
+    public class TrackerEndpointResponse : Enumeration, ITrackerEndpointDataModel
+    {
+        public static readonly TrackerEndpointResponse Success =
+            new TrackerEndpointResponse(1, nameof(Success));
+
+        public static readonly TrackerEndpointResponse UnexpectedException =
+            new TrackerEndpointResponse(-1, nameof(UnexpectedException));
+

```

Comment:
I renamed TrackerEndpointErrorResponse to TrackerEndpointResponse so it could include the Success response that this action returns and the switch statement will still work. Not sure if there's a better way to do that